kafka-ui-api/src/main/java/com/provectus/kafka/ui/serde/schemaregistry/SchemaRegistryAwareRecordSerDe.java

```diff
@@ -66,11 +68,25 @@ private static SchemaRegistryClient createSchemaRegistryClient(KafkaCluster clus
     List<SchemaProvider> schemaProviders =
         List.of(new AvroSchemaProvider(), new ProtobufSchemaProvider(), new JsonSchemaProvider());
     //TODO add auth
+
+    Map<String, String> configs = new HashMap<>();
+    if (cluster.getSchemaRegistry().isBasicAuthEnabled()) {
```

Comment:
could we check only user & pass fields? why do we need additional boolean for this?